workspaces/azure-devops/.changeset/healthy-wolves-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-scaffolder-backend-module-azure-devops': minor
+---
+
+Adds support for templateParameters: The createAzureDevopsRunPipelineAction action has been updated to accept and pass templateParameters to Azure Pipelines when triggering a pipeline run. This enhancement allows users to pass custom template parameters to Azure DevOps pipelines from Backstage.
```

Comment:
```suggestion
Adds support for template parameters to the `azure:pipeline:run` action
```

Reply:
I updated the changeset as per your suggestion

Reply:
I also updated the README.md